cmd/influxd/launcher/launcher.go

```diff
@@ -65,6 +68,127 @@ const (
 	JaegerTracing = "jaeger"
 )
 
+func NewCommand() *cobra.Command {
+	l := NewLauncher()
+	cmd := &cobra.Command{
+		Use:   "run",
+		Short: "Start the influxd server (default)",
+		Run: func(cmd *cobra.Command, args []string) {
+			// exit with SIGINT and SIGTERM
+			ctx := context.Background()
+			ctx = signals.WithStandardSignals(ctx)
+
+			if err := l.run(ctx); err != nil {
+				fmt.Fprintln(os.Stderr, err)
+				os.Exit(1)
+			} else if !l.Running() {
+				os.Exit(1)
+			}
+
+			var wg sync.WaitGroup
+			if !l.ReportingDisabled() {
+				reporter := telemetry.NewReporter(l.Registry())
+				reporter.Interval = 8 * time.Hour
+				reporter.Logger = l.Logger()
+				wg.Add(1)
+				go func() {
+					defer wg.Done()
+					reporter.Report(ctx)
+				}()
+			}
+
+			<-ctx.Done()
+
+			// Attempt clean shutdown.
+			ctx, cancel := context.WithTimeout(ctx, 2*time.Second)
+			defer cancel()
+			l.Shutdown(ctx)
+			wg.Wait()
+		},
+	}
+
+	buildLauncherCommand(l, cmd)
+
+	return cmd
+}
+
+func buildLauncherCommand(l *Launcher, cmd *cobra.Command) {
+	dir, err := fs.InfluxDir()
+	if err != nil {
+		panic(fmt.Errorf("failed to determine influx directory: %v", err))
+	}
+
+	opts := []cli.Opt{
+		{
+			DestP:   &l.logLevel,
+			Flag:    "log-level",
+			Default: "info",
```

Comment:
I wouldn't mind `zapcore.InfoLevel.String()` but not important.

Reply:
Done